ptime.go

```diff
@@ -1250,3 +1250,11 @@ func getWeekday(wd time.Weekday) Weekday {
 func (t *Time) resetWeekday() {
 	t.wday = getWeekday(t.Time().Weekday())
 }
+
+func (t Time) IsFuture() bool {
```

Comment:
It is better to name the method [`After`](https://pkg.go.dev/time#Time.After) and use a compatible signature to ensure alignment with the standard `time` package.